ethers-core/src/types/transaction/eip2930.rs

```diff
@@ -95,6 +95,7 @@ pub enum Eip2930RequestError {
 pub struct Eip2930TransactionRequest {
     #[serde(flatten)]
     pub tx: TransactionRequest,
+    #[serde(rename = "accessList")]
```

Comment:
i don't know if it was switching to anvil or what but typed_txs test failed sending the transaction with "unknown field" because this was not camelCase